CMakeLists.txt

```diff
@@ -142,6 +142,10 @@ message(" ")
 
 include(SSGCommon)
 
+# Build shared CPE first
```

Comment:
The targets won't be built in the order in which they are declared. You need explicit deps.